src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/RoadNodesToZipArchiveWriter.cs

```diff
@@ -28,8 +30,7 @@ public RoadNodesToZipArchiveWriter(RecyclableMemoryStreamManager manager, Encodi
         if (request == null) throw new ArgumentNullException(nameof(request));
         if (context == null) throw new ArgumentNullException(nameof(context));
 
-        var nodes =
-            await context.RoadNodes.InsideContour(request.Contour).ToListAsync(cancellationToken);
+        var nodes = await context.RoadNodes.ToListWithPolygonials(request.Contour, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken);
```

Comment:
use a line-separated method chain (like what it was)

Reply:
Done